test/unit/email_rate_limit_test.rb

```diff
@@ -2,7 +2,8 @@
 
 class EmailRateLimitTest < ActiveSupport::TestCase
   def seed_array
-    @seed_array ||= (1..350).to_a # max value has to be divisible by the multiplier of th higest used value i.e. 10_000/30.0
+    # max value has to be divisible by the multiplier of th higest used value i.e. 10_000/30.0
```

Comment:
This file just has line length changes (apologies for being difficult like that)